### ISSUE TITLE
`BiomeMobWeight.csv` isn't being recognized

### ISSUE THREAD

**Issue by thimblebird** [mactso/SpawnBalanceUtility#14]
[\config\spawnbalanceutility\BiomeMobWeight.csv](https://github.com/mactso/SpawnBalanceUtility/files/10148824/BiomeMobWeight.csv)
[latest.log](https://github.com/mactso/SpawnBalanceUtility/files/10148830/latest.log)

**Comment by thimblebird** [comment]
oh, it's on my client on windows 10. what do you mean with "try"? did a search and couldn't find it

**Comment by mactso** [comment]
I looked on our server and it is capitalized the way you have it, "BiomeMobWeight.csv"
Odd, in the file I downloaded from you one of the biomes had the word "try" in it.
Let's see.. did you save the .csv in the spawnbiomeutility subdirectory with the BiomeMobWeight.txt output file?  That's where SBU is looking for it.

**Comment by thimblebird** [comment]
Yep, didn't change the file location. Only thing I can think of at the top of my head may be some file encoding shenanigans, if that is even possible or an issue. Otherwise, I just renamed the *.txt file to *.csv and opened/edited it with Excel 365. It did work at some point, then suddenly didn't after editing the file a couple of times.

**Comment by mactso** [comment]
There may be some kind of unexpected errors in the file.

Try and rename your work file out of the way and recreate it as the .txt and rename that to .csv.   If it finds it, then start adding your work back in.   I did see that one line where a biome category was the word "try" which was odd but I don't think I check the category- it's just an informational string.

Anyway, add back in your work a section/biome/several biomes at a time.  That would help you locate the broken section.  Also, it may be printing an error to the console or outputting an error to your log.

**Comment by mactso** [comment]
Wait.. that wasn't "try"  That was "icy" combined with my astigmatism and cataract.

**Comment by mactso** [comment]
It's only 11,000 lines long.  I've had up to 21,000 so the size is fine.

**Comment by mactso** [comment]
Going to see about loading it into my test environment.  I don't have the mods but I should see any basic failure to see the file.

**Comment by mactso** [comment]
Okay the file read fine without any errors.
All I did was save the copy you posted as a csv text file from LibreOffice.

**Comment by mactso** [comment]
You might want to load and save it with LibreOffice.   Libreoffice is free.

**Comment by mactso** [comment]
Back up all your work of course!

**Comment by mactso** [comment]
![image](https://user-images.githubusercontent.com/9662494/206352355-7e45d16c-af5f-4cf7-8f43-c25910f771ab.png)

**Comment by mactso** [comment]
So I saved a raw copy from your issue and a copy from libre office and compared them with "WinMerge" and the files are identical.

The only other thing I can think of is renaming the copy on your disk out of the way and downloading the copy in this issue to the correct name and see if it loads.

What version of minecraft are you running?   I can upload a version with the stack trace in that error enabled.

**Comment by mactso** [comment]
Okay, lupicus may have figured this out.
You removed all mobs from one of the biomes.
SBU is running correctly but SBU is incorrectly printing the file not found error since SBU can't find any mobs for that biome.  You can suppress the error by adding at least 1 mob (bats?) into that biome until I account for that edge case.
